lib/utils/file.js

```diff
@@ -0,0 +1,17 @@
+// Based on https://github.com/kevva/executable
+// Since this module doesn't expose the function to check stat mode only,
+// his logic is pasted here.
+// Opened an issue and a pull request about it,
+// to maybe switch to module in the future:
+// Issue: https://github.com/kevva/executable/issues/9
+// PR: https://github.com/kevva/executable/pull/10
+
+export function isExecutable (fileStat) {
+  if (process.platform === 'win32') return true;
```

Comment:
I don't understand, sorry. You mean folders should not be `cd`ed?
The behaviour of iTerm.app in this case is to `cd` if it's a folder, or append the return to the path if it's a file (without checking if it's executable or not like I did).

Why it's inconsistent to you?